### PR TITLE
Replace fos user bundle

### PR DESCRIPTION
**Pull request by kennyva**

### CODE REVIEW

File: src/Migrations/Version20190813114803.php [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20190813114803 extends AbstractMigration
+{
+    public function getDescription() : string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema) : void
```
Comment (jimmycleuren) [inline src/Migrations/Version20190813114803.php]: can't we transfer the users to the new table ? Like this, nobody will be able to login anymore.

File: config/packages/security.yaml [hunk 2]
```diff
@@ -20,31 +29,28 @@ security:
...
-            logout:       true
-            anonymous:    true
+            guard:
+                authenticators:
+                    - App\Security\BasicAuthenticator
+            logout:
+                path: /logout
+                target: /
+                success_handler: App\Security\AfterLogoutRedirection
+            anonymous: true
```
Comment (Rubc) [inline config/packages/security.yaml]: What does this achieve?

File: config/packages/security.yaml [hunk 3]
```diff
@@ -20,31 +29,28 @@ security:
...
-                  provider: fos_userbundle
-                  csrf_token_generator: security.csrf.token_manager
-            logout:       true
-            anonymous:    true
+            guard:
+                authenticators:
+                    - App\Security\BasicAuthenticator
+            logout:
+                path: /logout
+                target: /
```
Comment (Rubc) [inline config/packages/security.yaml]: Do we need this? Logout function is not implemented

File: config/packages/security.yaml [hunk 4]
```diff
@@ -20,31 +29,28 @@ security:
...
-
-    # Easy way to control access for large sections of your site
-    # Note: Only the *first* access control that matches will be used
+            # activate different ways to authenticate http_basic: true
+            # https://symfony.com/doc/current/security.html#a-configuring-how-your-users-will-authenticate form_login:
+            # true https://symfony.com/doc/current/security/form_login_setup.html
+    # Easy way to control access for large sections of your site Note: Only the *first* access control that matches
+    # will be used
     access_control:
+         - { path: ^/login$, roles: IS_AUTHENTICATED_ANONYMOUSLY }
```
Comment (Rubc) [inline config/packages/security.yaml]: $?

File: src/Security/BasicAuthenticator.php [hunk 5]
```diff
@@ -0,0 +1,107 @@
+<?php
+
+
+// src/Security/LoginFormAuthenticator.php
```
Comment (Rubc) [inline src/Security/BasicAuthenticator.php]: Remove comment

**Comment by jimmycleuren** [comment]
Users don't have a way to create an initial user after installation now ? Maybe we should add a command to do that ?

**Pull request merged**